plugins/org.jboss.ide.eclipse.freemarker/src/org/jboss/ide/eclipse/freemarker/editor/ReconcilingStrategy.java

```diff
@@ -107,18 +107,19 @@ void reconcile() {
 			this.editor.updateModel(regions, stamp1 == stamp2 ? stamp1 : null);
 		}
 
-		ParseException e = checkTemplateSyntax();		
+		ParseException e = checkTemplateSyntax();
 		this.editor.updateMarkers(e);
 	}
 
 	private Configuration fmConfiguration;
-	
+
 	/**
 	 * Returns the syntactical error in the template, or {@code null} if there's none.
 	 */
 	private ParseException checkTemplateSyntax() {
 		if (fmConfiguration == null) {
-			fmConfiguration = new Configuration(Configuration.getVersion());
+			// see Freemarker configuration; we should use the version we are supporting, not the latest.
+			fmConfiguration = new Configuration(Configuration.VERSION_2_3_31);
```

Comment:
That comment is true for most application, but not for some development tools. So this should be `new Version(Configuration.getVersion())`. The copying is needed to avoid FreeMarker logging a WARN.